CONTRIBUTING.md

```diff
@@ -1 +1,72 @@
-# Contributing
+# Contributing Guidelines
+
+Welcome to the contributing guidelines for the Planet Of The Bugs repository on GitHub! We appreciate your interest in contributing to our project. Whether you're an experienced developer or just starting out, we value your input and contributions.
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Forking the Repository](#forking-the-repository)
+  - [Setting Up Your Local Environment](#setting-up-your-local-environment)
+- [Contributing](#contributing)
+  - [Creating a New Feature or Bugfix](#creating-a-new-feature-or-bugfix)
+  - [Making Changes](#making-changes)
+  - [Submitting a Pull Request](#submitting-a-pull-request)
+- [Code of Conduct](#code-of-conduct)
+
+## Getting Started
+
+Thank you for considering contributing to our project. Before you begin, please make sure you have a GitHub account. If you don't have one, you can sign up [here](https://github.com/signup).
+
+### Forking the Repository
+
+1. Navigate to the Planet Of The Bugs repository on GitHub: [https://github.com/Emmanuel-Melon/planet-of-the-bugs](https://github.com/Emmanuel-Melon/planet-of-the-bugs)
+2. Click the "Fork" button in the upper right corner of the repository page. This will create a copy of the repository in your GitHub account.
```

Comment:
The fork and PR workflow is the standard Github workflow, but we have been using a Branch-PR. The differences between the 2 aren't much and would depend on our preference. @Emmanuel-Melon what do u think?

Reply:
I'll let you decide here!